llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -530,34 +530,34 @@ void RISCVFrameLowering::emitPrologue(MachineFunction &MF,
     const RISCVRegisterInfo *RI = STI.getRegisterInfo();
     if (RI->hasStackRealignment(MF)) {
       Align MaxAlignment = MFI.getMaxAlign();
-      unsigned SPAddrReg;
+      unsigned SPAddrSrcReg = SPReg;
```

Comment:
Register while you're here? Presumably SPReg was still unsigned when I wrote the code.